wiki/en/en-Walk-Through.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: wiki
+title: "Jamulus Walk-Through"
+lang: "en"
+permalink: "/wiki/Walk-Through"
+---
+# Jamulus walk-through
+Let's have a look at what Jamulus looks like and what you should do after having installed the software.
+## First start
+Before you start Jamulus:
+1. Close all other applications
+2. Plugin everything (Ethernet cable, audio gear, etc.)
+
+... to minimise CPU load and to ensure that no other application tries to use your sound card.
```

Comment:
Not sure everyone knows what "CPU" means, so I'd just say "to minimise load on your computer".

Reply:
Done.